src/Assessment/Assess.cs

```diff
@@ -566,6 +569,26 @@ public bool BeginAssessment()
             return true;
         }
 
+        private void UpdateExchangeRates()
+        {
+            UserInputObj.LoggerObj.LogInformation("Obtaining exchange rates' data");
+            double exchangeRate = 1.0;

```

Comment:
How should we communicate back to the user that we are using USD incase of failures? Should we update the excel output with currency = USD along with warning message below?